src/functionalTest/resources/json/expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json

```diff
@@ -1 +1 @@
ame":"Yes"}},{"value":{"applyingExecutorName":"Edward David Miller","applyingExecutorEmail":"exec4@test.email.com","applyingExecutorAddress":{"AddressLine1":"Flat 30","AddressLine2":"Searles Road","County":"Middlesex","PostTown":"Hayes","PostCode":"LV2 8QB"},"applyingExecutorOtherNames":"Ted Mills","applyingExecutorHasOtherName":"Yes"}}],"executorsNotApplying":[{"value":{"notApplyingExecutorName":"Baker Bonds","notApplyingExecutorReason":"DiedBefore","notApplyingExecutorIsDead":"No","notApplyingExecutorDiedBefore":"Yes"}},{"value":{"notApplyingExecutorName":"Bob Bonds","notApplyingExecutorReason":"PowerReserved","notApplyingExecutorIsDead":"No","notApplyingExecutorDiedBefore":"No"}},{"value":{"notApplyingExecutorName":"Billy Bonds","notApplyingExecutorReason":"PowerOfAttorney","notApplyingExecutorIsDead":"No","notApplyingExecutorDiedBefore":"No"}}],"solsSolicitorAppReference":"","solsSolicitorFirmName":"","solsSolicitorIsApplying":"No","solsFeeAccountNumber":"","solsSOTName":"","solsSolicitorEmail":"","solsSolicitorPhoneNumber":"","scannedDocuments":[{"value":{"controlNumber":"123135453645","fileName":"123135453645.pdf","type":"form","subtype":"PA1P","scannedDate":"2019-07-15T12:34:56.789Z","url":{"document_url":"https://dm-store:5006/fdfsfsdf","document_binary_url":"https://dm-store:5006/fdfsfsdf","document_filename":"123135453645.pdf"},"exceptionRecordReference":"1000002","deliveryDate":"2019-07-18T12:00:00.000Z"}}],"adopted":"Yes","adoptiveRelatives":[{"value":{"name":"Tim Jones","relationship":"Son","adoptedInOrOut":"in"}},{"value":{"name":"Bob Jones","relationship":"Brother","adoptedInOrOut":"out"}},{"value":{"name":"Mark Jones","relationship":"Cousin","adoptedInOrOut":"in"}},{"value":{"name":"Brian Jones","relationship":"Brother","adoptedInOrOut":"out"}},{"value":{"name":"Pat Jones","relationship":"Son","adoptedInOrOut":"in"}}],"spouseOrPartner":"Yes","applyingAsAnAttorney":"Yes","attorneyOnBehalfOfNameAndAddress":[{"value":{"name":"Neil Kinnock","address":{"AddressLine1":"Not Number 10","AddressLine2":"Seeing Red","AddressLine3":"","County":"Greater London","PostTown":"London Town","PostCode":"NW1 1LD","Country":""}}}],"mentalCapacity":"Yes","courtOfProtection":"No","epaOrLpa":"Yes","epaRegistered":"No","domicilityCountry":"","domicilityIHTCert":"Yes","willDate":"2013-06-20","willsOutsideOfUK":"Yes","willGiftUnderEighteen":"No","notifiedApplicants":"No","foreignAsset":"Yes","foreignAssetEstateValue":"2000000","primaryApplicantSecondPhoneNumber":"701231231231","applicationFeePaperForm":"21500","feeForCopiesPaperForm":"2000","totalFeePaperForm":"23500","paperPaymentMethod":"cheque","paymentReferenceNumberPaperform":"CHK-REF-123456","dateOfDivorcedCPJudicially":"2015-06-01","courtOfDecree":"Manchester","bulkScanCaseReference":"1000002","outsideUKGrantCopies":1,"childrenOverEighteenSurvived":"2","childrenUnderEighteenSurvived":"1","grandChildrenSurvivedUnderEighteen":"0","grandChildrenSurvivedOverEighteen":"2","caseType":"gop"}},"warnings":[]}
```

Comment:
make sure no other tests are using this/break if you add anything to the payload

Reply:
All functional tests passed and expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json isn't used anywhere else